src/help_parser/src/lib.rs

```diff
@@ -0,0 +1,207 @@
+pub struct HelpParser;
```

Comment:
Why is it a struct? It's just some free functions right?

Reply:
Also this file is missing LICENSE info

Reply:
I made it a struct to show that the functions together represent the concept of a "parser". Plus it is more convenient to include a single struct instead of three functions.

I added the license header.

Reply:
I'd agree with you if the parser had any fields, but as it stands, it's just confusing. Also, you don't have to import the functions, because you can just qualify them with module.

What we could do is make the parser contain the markdown contents. Something like:

```rust
let parser = HelpParser::new("some_filename.md");
parser.usage();
```

But that might also not be super convenient.

Reply:
Ok, I removed the struct in the latest push.